runtime/dart_init.cc

```diff
@@ -396,10 +396,8 @@ Dart_Isolate IsolateCreateCallback(const char* script_uri,
 
     if (!kernel_data.empty()) {
       // We are running kernel code.
-      uint8_t* kernel_buf = static_cast<uint8_t*>(malloc(kernel_data.size()));
-      memcpy(kernel_buf, kernel_data.data(), kernel_data.size());
-      FXL_CHECK(!LogIfError(Dart_LoadKernel(Dart_ReadKernelBinary(
-          kernel_buf, kernel_data.size(), ReleaseFetchedBytes))));
```

Comment:
Perhaps this function can be removed too?

Reply:
There's one use left (Dart_ReadKernelBinary for platform.dil).